actions/installer/README.md

```diff
@@ -0,0 +1,37 @@
+# slsa-verifier setup GitHub Action
+
+This action installs the SLSA verifier and adds it to your PATH.
+
+For more information about `slsa-verifier`, refer to [its documentation](https://github.com/slsa-framework/slsa-verifier#verification-of-provenance).
+
+For more information about SLSA in general, see [https://slsa.dev](https://slsa.dev).
+
+## Usage
+
+To install the latest version, use:
+
+```yaml
+uses: slsa-framework/slsa-verifier-installer
+```
+
+To install a specific version, use:
+```yaml
+uses: slsa-framework/slsa-verifier-installer
+  with:
+    verifier-version: v1.3.0
+```
+
+NB: Pinning this action to a relase tag or commit SHA will pin you to the latest version of `slsa-verifier` at the time of release.
+
+### Optional Inputs
+
+| Input | Description |
+| --- | --- |
+| `verifier-release` | `slsa-verifier` version to use. Default value is `latest`. |
+
+See https://github.com/slsa-framework/slsa-verifier/releases for the list of available `slsa-verifier` releases.
+
+For a full example workflow, see `.github/workflows/test_installer.yml`.
```

Comment:
nit: consider making this a link? In order to be a relative URL I guess it would need to be with `../../.github/workflows/test_installer.yml`